### ISSUE TITLE
Add tag to render the react-refresh script

### ISSUE THREAD

**Issue by mixxorz** [MrBin99/django-vite#15]
I wanted to raise another suggestion for django-vite.

When working with `@vitejs/plugin-react-refresh`, we need to include this script to enable HMR.

```html
<script type="module">
  import RefreshRuntime from 'http://localhost:3000/@react-refresh'
  RefreshRuntime.injectIntoGlobalHook(window)
  window.$RefreshReg$ = () => {}
  window.$RefreshSig$ = () => (type) => type
  window.__vite_plugin_react_preamble_installed__ = true
</script>
```

It would be fantastic if we could have a `{% vite_react_refresh_tag %}` or something similar to render this script. It would only render the script if `DEBUG` is true.

Thanks again for your wonderful work on this library.

Ref: https://vitejs.dev/guide/backend-integration.html

**Comment by MrBin99** [comment]
Duplicate of #14

**Comment by burnedikt** [comment]
I think this has been done with #53 but not released yet?

**Comment by burnedikt** [comment]
> I think this has been done with #53 but not released yet?

Seems like it is released now in [version 2.1.0](https://github.com/MrBin99/django-vite/releases/tag/2.1.0) 🎉
